docs/app_modules/interchain-accounts/ica_auth.md

```diff
@@ -0,0 +1,219 @@
+<!--
+order: 3
+-->
+
+# Building an ICA Authentication Module
+
+The controller module is used for account registration and packet sending. 
+It executes only logic required of all controllers of interchain accounts. 
+The type of authentication used to manage the interchain accounts remains unspecified. 
+There may exist may different types of authentication which are desirable for different use cases. 
+Thus the purpose of the authentication module is to wrap the controller module with custom authentication logic.
+
+In ibc-go, authentication modules are connected to the controller chain via a middleware stack. 
+The authentication module acts as the base application within this stack. 
+To implement an authentication module, the `IBCModule` interface must be fulfilled. 
+By implementing the controller module as middleware, any amount of authentication modules can be created and connected to the controller module without writing redundant code. 
+
+The authentication module must:
```

Comment:
Should we add claiming of channel capability to this bulleted list? I know it is mentioned below but just a thought as I read through.

But maybe it's okay to omit(?) if this intended to be a higher level overview. Your call!